fission/src/systems/physics/PhysicsSystem.ts

```diff
@@ -658,6 +661,91 @@ class PhysicsSystem extends WorldSystem {
         }
     }
 
+    /**
+     * Creates the Jolt ShapeSettings for a given part using the Part Definition of said part.
+     *
+     * @param   partDefinition  Definition of the part to create.
+     * @returns If successful, the created convex hull shape settings from the given Part Definition.
+     */
+    private CreateConcaveShapeSettingsFromPart(
+        partDefinition: mirabuf.IPartDefinition
+    ): [Jolt.ShapeSettings, Jolt.Vec3, Jolt.Vec3] | undefined | null {
+        const settings = new JOLT.MeshShapeSettings()
+
+        settings.mMaxTrianglesPerLeaf = 8
+
+        settings.mTriangleVertices = new JOLT.VertexList()
+        settings.mIndexedTriangles = new JOLT.IndexedTriangleList()
+        settings.mMaterials = new JOLT.PhysicsMaterialList()
+
+        settings.mMaterials.push_back(new JOLT.PhysicsMaterial())
+
+        const min = new JOLT.Vec3(1000000.0, 1000000.0, 1000000.0)
+        const max = new JOLT.Vec3(-1000000.0, -1000000.0, -1000000.0)
```

Comment:
```suggestion
        const min = new JOLT.Vec3(Number.POSITIVE_INFINITY, Number.POSITIVE_INFINITY, Number.POSITIVE_INFINITY)
        const max = new JOLT.Vec3(Number.NEGATIVE_INFINITY, Number.NEGATIVE_INFINITY, Number.NEGATIVE_INFINITY)

```

Reply:
On second look it appears that these are the bounds, for some reason I thought that this was for finding the max and min. Not sure why I thought that. If the purpose is to set the bound restriction to these "reasonable" numbers and not necessary infinity then feel free to dismiss this suggestion.

Reply:
Infinity is fine